src/plugins/console/server/routes/api/console/autocomplete_entities/index.ts

```diff
@@ -43,7 +43,10 @@ const getAliases = async (settings: SettingsToRetrieve, esClient: IScopedCluster
 
 const getDataStreams = async (settings: SettingsToRetrieve, esClient: IScopedClusterClient) => {
   if (settings.dataStreams) {
-    const dataStreams = await esClient.asCurrentUser.indices.getDataStream();
+    const dataStreams = await esClient.asCurrentUser.indices.getDataStream({
+      name: '*',
+      expand_wildcards: 'all',
```

Comment:
So was this previously loading dataStreams via the api but not using it in the client?

Reply:
Outside the scope of this PR - I bet loading all indices / data streams is VERY slow in large clusters.

Reply:
> So was this previously loading dataStreams via the api but not using it in the client?

Yes, we didn't really use the data streams route.

> Outside the scope of this PR - I bet loading all indices / data streams is VERY slow in large clusters.

Yes, it might be... We have an open issue for performance testing for the autocompletion but haven't gotten to it yet: https://github.com/elastic/kibana/issues/151160